changelogs/fragments/74061-iptables-unit_test-fix_comment.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

Comment:
This isn't really a [minor change](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). Since this is just changing a comment in a unit test, you can omit the changelog for this. Thank you for your contribution.

Reply:
I wasn't sure, thanks for clarifying. Reverted in https://github.com/ansible/ansible/pull/74061/commits/3da0226d5cb85eb839e29b5813084895e8dbb0f7.